src/containers/PageConstructor/components/ConstructorBlock/ConstructorBlock.tsx

```diff
@@ -30,7 +31,8 @@ export const ConstructorBlock: React.FC<WithChildren<ConstructorBlockProps>> = (
         [data],
     );
 
-    const {top, bottom} = indent || {top: 'l', bottom: 'l'};
+    const {top, bottom} =
+        indent || data.resetPaddings ? {top: '0', bottom: '0'} : {top: 'l', bottom: 'l'};
```

Comment:
sounds like we should move all the `indent` logic to `BlockBase` component and handle it like other base block properties like `anchor` or `resetPaddings`

Reply:
I agree